js/carousel.js

```diff
@@ -608,7 +623,7 @@
      * @param {Function} callback
      */
     _cycleTo(n, callback) {
-      let diff = this.center % this.count - n;
+      let diff = (this.center % this.count) - n;
```

Comment:
This function can cause divide by zero when this.count equals zero

Reply:
hmm I haven't introduced this line... the linter added parenthesis, but the operation is there since 20 apr 2018.

Should we rather create another issue and accept this PR ?

Reply:
I haven't introduced this line, it's there since Apr. 2018. It was edited automatically by my code editor's linter.
Should we rather create another issue and accept this PR ?

Reply:
I think it should be fine because it is modulo and not divide operator.